### DISCUSSION TITLE
Supabase Realtime query not working

### DISCUSSION THREAD

**Discussion by rahul7400** [supabase/supabase#1975] [Questions] [answered]
I have written a simple code to subscribe a particular row in supabase

```
 const mySubscription = supabase 
  .from('Users : userid = eq.someid')
    .on('UPDATE',payload=>{
        console.log(payload)
    })
    .subscribe()  
```
**But the above code is not giving me any response but if i change my code like below -** 

```
 const mySubscription = supabase 
  .from('Users')
    .on('UPDATE',payload=>{
        console.log(payload)
    })
    .subscribe()  
```
that is I have changed the value of "**FROM**" to only my table name then i get the realtime response.
**why is the query not working as i want to subscribe to a particular row only?**

**Comment by cdedreuille** [comment, marked as answer]
Can you try removing spaces in `.from()`? Also perhaps check if the table `Users` uses a capital letter or not.

```
const mySubscription = supabase
  .from('Users:userid=eq.someid')
  .on('UPDATE', handleRecordUpdated)
  .subscribe()
```

**Reply by rahul7400** [reply]
hey @cdedreuille i checked both the things. Still no response
